subsys/bluetooth/host/conn.c

```diff
@@ -1014,7 +1024,7 @@ void bt_conn_suspend_tx(bool suspend)
 {
 	_suspend_tx = suspend;
 
-	LOG_DBG("%sing all data TX", suspend ? "suspend" : "resum");
+	LOG_DBG("%sing all data TX", suspend ? "suspend" : "resume");
```

Comment:
I don't think this change is correct, if you look at what suffix the format string is combining the string with to form a word. IMO this has been made unnecessarily "optimized" and the string should just have `%s` and the parameters would then be "suspending" and "resuming".

Reply:
Right, good catch. My IDE just reported it as a typo, but I see that it's not.